### PR TITLE
Updating Operations API to support Python's AutoRest

### PR DESCRIPTION
**Pull request by sonathan**
This checklist is used to make sure that common issues in a pull request are addressed. This will expedite the process of getting your pull request merged and avoid extra work on your part to fix issues discovered during the review process. 

### PR information
- [x] The title of the PR is clear and informative.
- [x] There are a small number of commits, each of which have an informative message. This means that previously merged commits do not appear in the history of the PR. For information on cleaning up the commits in your pull request, [see this page](https://github.com/Azure/azure-powershell/blob/dev/documentation/cleaning-up-commits.md).
- [x] Except for special cases involving multiple contributors, the PR is started from a fork of the main repository, not a branch.
- [x] If applicable, the PR references the bug/issue that it fixes.
- [ ] Swagger files are correctly named (e.g. the `api-version` in the path should match the `api-version` in the spec).

### Quality of Swagger
- [ ] I have read the [contribution guidelines](https://github.com/Azure/azure-rest-api-specs/blob/master/.github/CONTRIBUTING.md).
- [ ] My spec meets the review criteria:
  - [ ] The spec conforms to the [Swagger 2.0 specification](https://github.com/OAI/OpenAPI-Specification/blob/master/versions/2.0.md).
  - [ ] The spec follows the guidelines described in the [Swagger checklist](../documentation/swagger-checklist.md) document.
  - [ ] [Validation tools](https://github.com/Azure/azure-rest-api-specs/blob/master/documentation/swagger-checklist.md#validation-tools-for-swagger-checklist) were run on swagger spec(s) and have all been fixed in this PR.

### CODE REVIEW

**Comment by sonathan** [comment]
@veronicagg Here is the updated PR for following: https://github.com/Azure/azure-rest-api-specs/pull/1453

File: specification/recoveryservices/resource-manager/Microsoft.RecoveryServices/2016-06-01/vaults.json [hunk 1]
```diff
@@ -419,90 +419,78 @@
...
-          "readOnly": true
+          "description": "Localized display name",
+          "type": "string"
         },
         "blobDuration": {
-          "description": "Blob duration.",
-          "format": "date-time",
-          "type": "string",
-          "readOnly": true
+          "description": "blob duration",
```
Comment (veronicagg) [inline specification/recoveryservices/resource-manager/Microsoft.RecoveryServices/2016-06-01/vaults.json]: @sonathan per our conversation regarding this property, should thedescription be updated to include more info on what the field is expected to look? Per your comment in the referred, closed PR:
"As this field is in Response and Shoebox creates blobs in customer storage account every hour. The value PT1H indicates blobs per hour. "

**Review by veronicagg — CHANGES_REQUESTED** [review]
Thanks @sonathan for re-sending this PR. 
We had already reviewed all content, so looks good. 
Just one left over question from the other PR, which I added inline.
After confirmation on that, it can be approved and merged. 
Thanks!

**Comment by veronicagg** [comment]
Updating assignment, since I already reviewed this PR, this was just reopening it.

File: specification/recoveryservices/resource-manager/Microsoft.RecoveryServices/2016-06-01/vaults.json (hunk 1 above)
Comment (sonathan) [reply specification/recoveryservices/resource-manager/Microsoft.RecoveryServices/2016-06-01/vaults.json]: Thanks for pointing this out. Updated now.

**Comment by sonathan** [comment]
@veronicagg Please let me know if there are still any pending changes that needs to be done before merging. Thanks a lot.

**Review by veronicagg — APPROVED** [review]

**Pull request merged**